src/__tests__/Cloud-test.js

```diff
@@ -229,7 +229,7 @@ describe('CloudController', () => {
         value: 12,
         when: { __type: 'Date', iso: '2015-01-01T00:00:00.000Z' },
       },
-      { useMasterKey: true },
+      { returnStatus: true, useMasterKey: true },
```

Comment:
Why shouldn't the method always return the status? It's currently also not an option.

Reply:
It always returns the status. This is a test to show that the option was set.

Reply:
I see now, the `returnStatus` is an existing option, I thought it was a new one.